gradle.properties

```diff
@@ -1,8 +1,8 @@
 projectVersion=1.1.2-SNAPSHOT
 micronautVersion=3.3.0
 micronautTestVersion=2.3.7
-groovyVersion=3.0.9
-spockVersion=2.0-groovy-3.0
+groovyVersion=3.0.15
```

Comment:
nitpick: Can you revert those as well? I think we should really reduce to the minimal fix, to make things easier.

Reply:
done